common/src/main/scala/org/neo4j/spark/util/Neo4jOptions.scala

```diff
@@ -301,6 +303,7 @@ case class Neo4jDriverOptions(
     if (acquisitionTimeout > -1) builder.withConnectionAcquisitionTimeout(acquisitionTimeout, TimeUnit.MILLISECONDS)
     if (livenessCheckTimeout > -1) builder.withConnectionLivenessCheckTimeout(livenessCheckTimeout, TimeUnit.MILLISECONDS)
     if (connectionTimeout > -1) builder.withConnectionTimeout(connectionTimeout, TimeUnit.MILLISECONDS)
+    if (fetchSize > -1) builder.withFetchSize(fetchSize)
```

Comment:
beware that `-1` is actually a valid value for the Java driver.
> By default, the value is set to 1000. Use -1 to disables back pressure and config client to pull all records at once after each run.

Reply:
Done